lib/clusterable/holding.rb

```diff
@@ -94,10 +94,18 @@ def self.table
       Services.holdings_table
     end
 
+    # apply order and used paged each -- fetches blocks of rows
+    # by doing e.g. select * from holdings where uuid > LAST_UUID order by uuid limit 1000
+    def self.paged_each(dataset,&block)
```

Comment:
The mysql2 driver also supports a 'streaming' option; the trilogy driver doesn't. Probably worth comparing overall performance there.